File: MPI/mpicheck/tests/functions/3.0/MPI_Comm_create_group.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    MPI_Group var_1;
    int var_2;
    MPI_Comm *var_3;
    int ret;
    /* calls */
    ret = MPI_Comm_create_group(var_0, var_1, var_2, var_3);
    ret = PMPI_Comm_create_group(var_0, var_1, var_2, var_3);
    return 0;
}

File: MPI/mpicheck/tests/functions/3.0/MPI_Comm_create_group.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4
        call mpi_comm_create_group(var_0, var_1, var_2, var_3, var_4)
       call pmpi_comm_create_group(var_0, var_1, var_2, var_3, var_4)
        end program main
    